collection-spec/collection-spec.md

```diff
@@ -161,6 +162,21 @@ Oftentimes it is possible to model data and assets with either a Collection or a
 Items as much as is feasible, as they designed for assets. Using Collection-level assets should only be used if there is not another
 option.
 
+#### item assets
+
+This serves two purposes:
+
+1. Provide a human-readable definition of assets available in any Items 
+   belonging to this Collection so that the user can determine the key(s) 
+   of assets they are interested in.
+2. Provide a way to programmatically determine what assets are available 
+   in any member Item. Otherwise a random Item needs to be examined to 
+   determine assets available, but a random Item may not be representative of the set.
+
+An Item Asset Object defined at the Collection level is nearly the same as the 
+[Asset Object in Items](../item-spec/item-spec.md#asset-object), except for two differences.
+The `href` field is not required, because Collections don't point to any data by themselves, but at least two other fields must be present.
```

Comment:
```suggestion
An Item Asset Object defined at the Collection level is nearly the same as the 
[Asset Object in Items](../item-spec/item-spec.md#asset-object), except for two differences.
The `href` field is not required, because Item Asset Definitions don't point to any data by themselves, but at least two other fields must be present.
```